tdrs-backend/tdpservice/settings/common.py

```diff
@@ -271,10 +271,10 @@ class Common(Configuration):
 
     # Django Rest Framework
     DEFAULT_RENDERER_CLASSES = ['rest_framework.renderers.JSONRenderer']
-    TEST_REQUEST_RENDERER_CLASSES = ['rest_framework.renderers.JSONRenderer']
+    TEST_REQUEST_RENDERER_CLASSES = ['rest_framework.renderers.JSONRenderer',
+                                     'rest_framework.renderers.MultiPartRenderer']
     if DEBUG:
```

Comment:
Ah, so we have DEBUG set in raft-tech circleci and hhs maybe doesn't?

Reply:
It seems so. But adding this renderer doesnt/shouldnt harm anything.